### PR TITLE
FW reconciliation with jump drive.

### PR DESCRIPTION
**Pull request by jafdy**
Basis of what I did in #998 before I broke the pull.

### CODE REVIEW

**Comment by Amazinite** [comment]
I thought that "to offer, require <outfit>" doesn't work.

**Comment by jafdy** [comment]
I have set it up correctly. It is actually used in the exact way suggested in the wiki. 
I am getting errors from it but my assumption was that the `require` condition is post 0.9.0. I haven't worked out how to compile the code yet.

**Comment by Amazinite** [comment]
Nvm, i was thinking of something else.

**Comment by jafdy** [comment]
That should work now. The Quarg branch has a `never` argument as I can't think of a way to allow it to trigger if you could have a Quarg drive.
It's as done as it can be from my perspective.

**Comment by Isaacssv552** [comment]
What was wrong with checking for a skylance, nanotech battery, or other quarg outfit?

**Comment by Isaacssv552** [comment]
Suggested Improvements to Quarg Option: (Fixes typos and improves clarity. -- Denotes comment.)
Uproar instantly breaks out. The speaker spends several moments shouting at everyone to be quiet before a semblance of order is regained.
One of the Dirt Belt representative gets up, "We’ve heard rumours of raids against the Quarg, but we never believed that it could be true."
A representative from Shangri-La stands saying "Another species with Jump Drives, called the Korath, has been raiding our planet, maybe Captain <last> has got Korath tech instead." 
Now a Paradise planet representative speaks, "Implying that you have defeated the Quarg is just an attempt to scare us into compliance!" This raises a "Hear, Hear." from the other Paradise planet reps.`
choice
				`	"Fine, pretend it was the Korath."`
					goto last
				`	"Charge the Skylance. Fire at will!"` --This is rather silly, just a joke.
					die
				`	"If I didn’t loot a Quarg ship how did I get the Skylances I killed the Navy with?"`
					goto "quarg weapons" --Haven't actually written this yet. Not sure how to deal with a player that has been massacring the Republic with Quarg tech.

**Comment by jafdy** [comment]
one Quarg outfit is as good as another but the `require` condition won't work in a `branch` so I have no way of using that branch at the moment.

**Comment by Isaacssv552** [comment]
If my attempt at adding a new tag goes well I might try and look at the mission interpreter and see if I can add support for "require", or whatever else makes sense. Unfortunately, just adding a tag is pushing the limits of my current c++ knowledge, so It'll probably be a while.

File: data/free worlds reconciliation.txt [hunk 1]
```diff
@@ -1554,6 +1558,610 @@ mission "FW Pug 3C"
...
+			`	"I know why." you interrupt. The silence this statment leaves behind is deafening.`
+			`	Evetuly Katya asks "<first>, is it the Syndicate?"`
+			`	"No, it is an alien race called the Pug. Thay say thay are there to promote peace, but I don't know their true intentions."`
+			`	JJ says, "The Senate had agreed that we must put our differences with the Republic aside, and meet with Parliament to deal with this crisis. But that was before you told us what has really happened."`
+			`	"Where did the Pug come from? And are they attacking Human ships?" Katya asks starting to sound worried.`
+			`	"They came from a system called Deneb, the star out beyond Zeta Aquilae. They didn't attack my ship when I arrived and nothing on the ground looked like it had been damaged." At this Katya looks relived. "There are some people who are resisting the Pug but there wasn't a single Free Worlds ship in orbit or on the ground so they were probably destroyed when the Pug invaded."`
+			`	Alondo contemplates this before speaking up "We can easily stop are own people panicking but we can't use the Net to tell the Republic, they just won't believe it."`
```
Comment (Waladil) [inline data/free worlds reconciliation.txt]: Line 1702, dialogue should read "We can easily stop our own people..." instead of "stop are own people..."

**Comment by jafdy** [comment]
Closing. Not well written, the discontinuity of proceeding with the current story is better.

**Pull request closed without merging**